core/src/test/java/org/hibernate/ogm/backendtck/elementcollection/DuplicatesAndOrderOfAssociationItemsTest.java

```diff
@@ -0,0 +1,147 @@
+		meeting.setName( NAME );
+
+		meeting.addAddress( ADDRESS );
+		meeting.addAddress( ADDRESS_ALT1 );
+		meeting.addAddress( ADDRESS_ALT2 );
+
+		meeting.addPhone( PHONE );
+		meeting.addPhone( PHONE );
+		meeting.addPhone( PHONE );
+
+		Programmer participant0 = new Programmer( PARTICIPANT_0 );
+		Programmer participant1 = new Programmer( PARTICIPANT_1 );
+		Programmer participant2 = new Programmer( PARTICIPANT_2 );
+
+		meeting.addParticipant( participant0 );
+		meeting.addParticipant( participant1 );
+		meeting.addParticipant( participant2 );
+
+		Programmer speaker = new Programmer( SPEAKER );
+		meeting.addSpeaker( speaker );
+		meeting.addSpeaker( speaker );
+		meeting.addSpeaker( speaker );
+
+		inTransaction( session -> {
+			session.save( participant0 );
+			session.save( participant1 );
+			session.save( participant2 );
+			session.save( speaker );
+			session.save( meeting );
+		} );
+
+		inTransaction( session -> {
+			Meeting load = session.load( Meeting.class, NAME );
+
+			// an embedded list with a defined @OrderColumn keeps duplicates
+			assertThat( load.getPhones() ).containsExactly( PHONE, PHONE, PHONE );
+
+			// without an @OrderColumn it may or may not keep duplicates
+			assertThat( load.getAddresses() ).containsOnly( ADDRESS, ADDRESS_ALT1, ADDRESS_ALT2 );
+
+			// a join table list with a defined @OrderColumn keeps duplicates
+			assertThat( load.getSpeakers() ).containsExactly( speaker, speaker, speaker );
+
+			// without an @OrderColumn it may or may not not keep duplicates
+			assertThat( load.getParticipants() ).containsOnly( participant0, participant1, participant2 );
+		} );
+	}
+
+	@Test
+	public void testPreserveOrderOnAssociationItems() {
+		Meeting meeting = new Meeting();
+		meeting.setName( NAME );
+
+		meeting.addAddress( ADDRESS );
+		meeting.addAddress( ADDRESS_ALT1 );
+		meeting.addAddress( ADDRESS_ALT2 );
+
+		meeting.addPhone( PHONE );
+		meeting.addPhone( PHONE_ALT1 );
+		meeting.addPhone( PHONE_ALT2 );
+
+		Programmer participant0 = new Programmer( PARTICIPANT_0 );
+		Programmer participant1 = new Programmer( PARTICIPANT_1 );
+		Programmer participant2 = new Programmer( PARTICIPANT_2 );
+
+		meeting.addParticipant( participant0 );
+		meeting.addParticipant( participant1 );
+		meeting.addParticipant( participant2 );
+
+		meeting.addSpeaker( participant0 );
+		meeting.addSpeaker( participant1 );
+		meeting.addSpeaker( participant2 );
+
+		inTransaction( session -> {
+			session.save( participant0 );
+			session.save( participant1 );
+			session.save( participant2 );
+			session.save( meeting );
+		} );
+
+		inTransaction( session -> {
+			Meeting load = session.load( Meeting.class, NAME );
+
+			// an embedded list with a defined @OrderColumn guarantees the order
+			assertThat( load.getPhones() ).containsExactly( PHONE, PHONE_ALT1, PHONE_ALT2 );
+
+			// without an @OrderColumn it doesn't guarantee the order
+			assertThat( load.getAddresses() ).containsOnly( ADDRESS, ADDRESS_ALT1, ADDRESS_ALT2 );
```

Comment:
> assertThat( load.getAddresses() ).containsOnly( ADDRESS, ADDRESS_ALT1, ADDRESS_ALT2 );

I don't think we need this test.
We already plenties of tests under `org.hibernate.ogm.backendtck.embeddable`.

For this issue I would change this to:

```
meeting.addAddress( ADDRESS );
meeting.addAddress( ADDRESS );
meeting.addAddress( ADDRESS );

...
assertThat( load.getAddresses() ).as(
"Hibernate OGM does not supporty duplicates for a collection of embeddebles (issue ...). If you've solved this issue, congratulations! Please, update the jira and this test accordingly and thanks a lot"
).containsOnly( ADDRESS  );
```